north/src/manifest.rs

```diff
@@ -122,12 +122,80 @@ pub struct CGroups {
 }
 
 // TODO: Remove?
-#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[derive(Clone, Debug, Eq, PartialEq)]
 pub struct Resource {
     pub name: Name,
     pub version: Version,
     pub dir: std::path::PathBuf,
-    pub mountpoint: std::path::PathBuf,
+}
+
+impl FromStr for Resource {
+    type Err = anyhow::Error;
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        let re = regex::Regex::new(r"(?P<name>\w+):(?P<version>[\d.]+)(?P<dir>[\w/]+)?")
```

Comment:
Move this into a lazy static to avoid the overhead to construct and parse the regex upon every manifest parsing run.